### ISSUE TITLE
RFE: add live-update ASM Signature - currently installed signature createDateTime metric (epoch)

### ISSUE THREAD

**Issue by amolari** [F5Networks/f5-telemetry-streaming#174]
### Is your feature request related to a problem? Please describe.
We would like to monitor our various ASM systems and be assured that a minimum version of the ASM signature has been installed

### Describe the solution you'd like
New metric derived from the system's currently installed ASM signature (live-update) createdDateTime.

On the CLI one can get the value with:

`curl -sku admin: https://localhost:<ssl-port>/mgmt/tm/live-update/asm-attack-signatures/installations | jq -r '.items[] | select(.status=="install-complete") | .updateFileReference.link' | sed 's/localhost/localhost:<ssl-port>/'  | xargs -n 1 curl -sku admin: | jq -r .createDateTime | date -f - +%s`

We first need to know which updateFile is the one currently installed, and then find the createDateTime value for that updateFile so 2 API reads are required.
Normalizing the returned value (ISO) to EPOCH  is prefered, as
- it's a true metric (numerical) and won't result as a tag
- it allows to directly perform some comparison operations on the consumer-side (such as if $value < xxxxxxx then,....)

### Additional context
UpdateFile from API-get example:
```
{
  "id": "ff8080817a39feb4017a39ffe7a4000f",
  "filename": "ASM-AttackSignatures_20210322_164818.im",
  "md5": "0A68994C7666A48B6DC2424D2A221B70",
  "fileLocationReference": {
    "link": "https://localhost/mgmt/tm/live-update/file-transfer/downloads/ASM-AttackSignatures_20210322_164818.im"
  },
  "createDateTime": "2021-03-22T16:48:18Z",
  "isFileAvailable": true,
  "isFileManuallyUploaded": true,
  "isGenesis": false,
  "kind": "tm:live-update:asm-attack-signatures:update-filesstate",
  "selfLink": "https://localhost/mgmt/tm/live-update/asm-attack-signatures/update-files/ff8080817a39feb4017a39ffe7a4000f"
}

```

**Comment by dstokesf5** [comment]
Thank you for your feedback. Your feature request has been added to our internal product backlog as AUTOTOOL-2969.

**Comment by dstokesf5** [comment]
This has been resolved in the 1.27.0 release.
